### PR TITLE
Docs changes

### PR DESCRIPTION
**Pull request by dangodev**
## Reason for change

This finishes up the fantastic work Helen did compiling all her [copy suggestions](https://docs.google.com/document/d/1IkdkZJASKIOnDpyMgekDjCyvmeCe1sS-hWVYwwFnxwA/edit). With that the issue can be closed.

Most of the edits are minor polish and syntax that aren’t worth discussing. The major change is here is in the sidebar navigation: 

Screen Shot 2019-10-17 at 12 24 10

`Components` has been renamed to `Styled Components`, `Data` has been renamed to `Unstyled Components`, and `Advanced` has been renamed to `Guides`. This doesn’t change the URLs, so bookmarks still work (for now).

## Checklist

- [ ] **CHANGELOG**: The **Unreleased** section of CHANGELOG was updated
- [x] **Prop changes**: [docs][docs] were updated
- [x] **Prop changes**: E2E tests were updated, testing from the highest level possible
- [ ] **Platform testing**: If this change should be [tested against a platform][platform-testing], has it been?

[docs]: https://ui.sandbox.manifold.co
[platform-testing]: https://app.gitbook.com/@manifold/s/engineering/playbooks/testing-your-code-against-a-platform

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -335,6 +346,13 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - Changed how the `manifold-service-card` works to have it fetch the product unless given. This
   allows it to be used standalone or in the `marketplace`.
 
+### Breaking changes
+
+- `resource-name` renamed to `resource-label` in the following components:
+  `<manifold-resource-list>`, `<manifold-resource-plan>`, `<manifold-resource-product>`,
+  `<manifold-data-product-logo>`, `<manifold-data-product-name>`,
+  `<manifold-data-provision-button>`, `<manifold-data-resource-list>`
+
```
Comment (dangodev) [inline CHANGELOG.md]: Breaking changes were taken out of docs and put into the CHANGELOG

File: docs/docs/advanced/theming.md [hunk 2]
```diff
@@ -283,72 +279,9 @@ Cards are used in Marketplace (service cards) and Plan Selector (plan buttons, p
...
-| <span class="tag not-implemented" /> `--sidebar-text-color-heading`     | `var(--manifold-text-color-heading)`   | Text color for sidebar headings. |
-| <span class="tag not-implemented" /> `--sidebar-font-family-heading`    | `var(--manifold-font-family-heading)`  | Font family for sidebar headings |
-| <span class="tag not-implemented" /> `--sidebar-font-weight-heading`    | `var(--manifold-font-weight-heading)`  | Weight for sidebar headings      |
-| <span class="tag not-implemented" /> `--sidebar-font-size-heading`      | `var(--manifold-font-size-h4)`         | Font size for sidebar headings   |
-|                                                                         |                                        |                                  |
-| <span class="tag not-implemented" /> `--sidebar-text-color-links`       | `var(--manifold-text-color-secondary)` | Text color for sidebar links.    |
-| <span class="tag not-implemented" /> `--sidebar-text-color-links-hover` | `var(--manifold-color-primary)`        | Text color for sidebar links.    |
```
Comment (dangodev) [inline docs/docs/advanced/theming.md]: Deleted a lot of “planning“ from the theming doc. I’d like to do a more in-depth survey of everything. This just cleans up info platforms can’t use for now.

File: docs/docs/components/manifold-credentials.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ example: |
 
 # 🔒 Credentials
 
-Display credentials for a resource. 🔒 Requires authentication.
```
Comment (dangodev) [inline docs/docs/components/manifold-credentials.md]: Authentication guide now linked.

File: docs/package.json [hunk 4]
```diff
@@ -34,21 +34,21 @@
...
-    "eslint-config-prettier": "^6.3.0",
-    "eslint-import-resolver-typescript": "^1.1.1",
+    "eslint-config-prettier": "^6.4.0",
+    "eslint-import-resolver-typescript": "^2.0.0",
     "eslint-plugin-prettier": "^3.1.1",
-    "eslint-plugin-react": "^7.15.1",
+    "eslint-plugin-react": "^7.16.0",
     "prettier": "^1.18.2",
-    "stylelint": "^11.0.0",
+    "stylelint": "^11.1.1",
```
Comment (dangodev) [inline docs/package.json]: Docs deps were updated, because dependabot doesn’t manage these.

File: CHANGELOG.md (hunk 1 above)
Comment (domenicrosati) [reply CHANGELOG.md]: thanks

File: docs/docs/advanced/theming.md (hunk 2 above)
Comment (domenicrosati) [reply docs/docs/advanced/theming.md]: k

**Pull request merged**